test/useTranslation.spec.js

```diff
@@ -53,14 +53,15 @@ describe('useTranslation', () => {
   });
 
   describe('few namespaces', () => {
-    function TestComponent() {
-      const { t, i18n } = useTranslation(['other', 'translation'], { i18n: i18nInstance });
-
+    it('hook destructured values are expected types', () => {
+      const { result } = renderHook(() =>
+        useTranslation(['other', 'translation'], { i18n: i18nInstance }),
+      );
+      const { t, i18n } = result.current;
       expect(typeof t).toBe('function');
       expect(i18n).toEqual(i18nInstance);
-
-      return <div>{t('key1')}</div>;
-    }
+      expect(<div>{t('key1')}</div>).toEqual(<div>key1</div>);
```

Comment:
does it have to wrapped in dev? can't we check the output `expect(t('key1')).toEqual('key1');`

Reply:
Jest said that was the `object` it was expecting when I tried other assertions. 🤷🏻‍♂️

Reply:
weird, we had done similar in other tests. See if that helps. If not this seems okay.

Reply:
No good. 
<img width="465" alt="Screen Shot 2020-12-07 at 9 38 42 AM" src="https://user-images.githubusercontent.com/27247160/101371121-02d5f100-3870-11eb-9264-db90eb8546da.png">

Reply:
Oh oops I forgot to remove the other Div lol